dataeng/jobs/analytics/LoadAffiliateWindowToWarehouse.groovy

```diff
@@ -0,0 +1,31 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.to_date_interval_parameter
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class LoadAffiliateWindowToWarehouse {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("affiliate-window") {
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters to_date_interval_parameter(allVars)
+            parameters {
+                stringParam('OUTPUT_SCHEMA', 'finance', '')
+            }
+            multiscm common_multiscm(allVars)
+            triggers common_triggers(allVars)
+            wrappers common_wrappers(allVars)
+            publishers common_publishers(allVars)
+            steps {
+                shell(dslFactory.readFileFromWorkspace('dataeng/resources/load-affiliate-window.sh'))
+                if (allVars.get('SNITCH')) {
```

Comment:
Don't bother with snitch, we don't use it anyway, and you don't have a SNITCH variable anyway.